test/Docker.DotNet.Tests/IContainerOperationsTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
```

Comment:
@Emdot nice job. 
Previous exception thrown was DockerContainerNotFound I suppose, could be more errors based on response HttpStatusCode or similar?
I might check this for some tests I did myself.

👍

Reply:
The API docs for the endpoint are wrong too, so presumably that's the source of the mistake.